index.js

```diff
@@ -22,7 +27,16 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
+
+    options.babel = options.babel || {};
+
+    if (process.env.EMBER_ENV === 'production') {
```

Comment:
Does this get set automatically by an `ember build --environment=production`? When testing locally I had to set it manually.